src/Compilers/CSharp/Portable/CommandLine/CSharpCommandLineParser.cs

```diff
@@ -1037,11 +1037,12 @@ public new CSharpCommandLineArguments Parse(IEnumerable<string> args, string bas
 
                         case "pathmap":
                             // "/pathmap:K1=V1,K2=V2..."
+                            unquoted = RemoveQuotesAndSlashes(value);
```

Comment:
Consider moving `unquoted = Remove...` into the block to match. Otherwise, block is confusing.